index.html

```diff
@@ -1,267 +1,256 @@
-          </p>
-          <ul class="dropdown-list">
-            <li>
-              <img src="images/js.svg" alt="" />
-              <span>Javascript</span>
-            </li>
-            <li>
-              <img src="images/html.svg" alt="" />
-              <span>HTML</span>
-            </li>
-            <li>
-              <img src="images/css.svg" alt="" />
-              <span>CSS</span>
-            </li>
-          </ul>
-        </div>
-        <div class="dropdown">
-          <p>
-            <span>Frameworks</span>
-            <img src="images/chevron-right.png" alt="" />
-          </p>
-        </div>
-        <div class="dropdown">
-          <p>
-            <span>Skills</span>
-            <img src="images/chevron-right.png" alt="" />
-          </p>
-        </div>
+      <div class="dropdown">
+        <p>
+          <span>Frameworks</span>
+          <img src="images/chevron-right.png" alt="" />
+        </p>
+      </div>
+      <div class="dropdown">
+        <p>
+          <span>Skills</span>
+          <img src="images/chevron-right.png" alt="" />
+        </p>
       </div>
-    </section>
-    
-    
-    <section class="contact--section">
-          <h2 class="contact--me">Contact me</h2>
-          <p class="contact--paragraph">
-            If you have an application you <br> are interested in developing,
-            a <br> feature that you need built or a <br> project that needs coding. <br>
-            I'd love to help with it!
-          </p>
-          <form action="https://formspree.io/f/xayzglpq" method="Post" class="contact--form">
-            <input type="text" name="name" id="name"
-                  required maxlength="30"
-                  placeholder="Write your name here">
-              <input type="email" name="email" id="email"
-                  required
-                  placeholder="Write your email here">
-                <textarea name="message" id="message" maxlength="500" required placeholder="write your message here"></textarea>
-          
-                <button type="submit" class="see--project contact-button">Get in touch</button>
+    </div>
+  </section>
+  <section class="contact--section">
+    <h2 class="contact--me">Contact me</h2>
+    <p class="contact--paragraph">
+      If you have an application you are interested in developing,
+      a feature that you need built or a project that needs coding.
+      I'd love to help with it!
+    </p>
+    <form action="https://formspree.io/f/xayzglpq" method="Post" class="contact--form">
+      <input type="text" name="name" id="name" required maxlength="30" placeholder="Write your name here">
+      <input type="email" name="email" id="email" required placeholder="Write your email here">
+      <textarea name="message" id="message" maxlength="500" required placeholder="write your message here"></textarea>
+      <button type="submit" class="see--project contact-button">Get in touch</button>
+    </form>
+  </section>
+</body>
```

Comment:
[OPTIONAL]
- [ ] Kindly stick to the `Figma` design specs as much as possible, and try to use the same `paddings` and `margins` as the `Figma` to reach the best look for your project.

| Your design | Figma Design |
| ------------- | ------------- |
| ![Screenshot 2023-06-11 001238](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/102612396/8da863e2-9003-4a3a-8065-60605d36cc40) | ![image](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/102612396/77f3d5d6-da02-4c3d-993d-e52cdab86b22) |